pkg/reporting/util.go

```diff
@@ -87,3 +94,62 @@ func isDevEndpoint(endpoint string) bool {
 	result, _ := regexp.MatchString(`replicated-app`, endpoint)
 	return result
 }
+
+func AirgapReportSecret(name string, namespace string, kotsadmUID apimachinerytypes.UID, data []byte) *corev1.Secret {
+	return &corev1.Secret{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Secret",
+		},
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
What do you think of adding the necessary labels for this to be included in snapshots?

Reply:
I think we should do it since rqlite data would have been restored

Reply:
I agree.